src/main/java/org/junitpioneer/jupiter/Util.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.lang.annotation.Annotation;
+
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.platform.commons.support.AnnotationSupport;
+
+/**
+ * Pioneer-internal utility class.
+ */
+class Util {
+
+	private Util() {
+		// private cvonstructor to prevent instantiation of utility class
+	}
+
+	/**
+	 * Determines whether an annotation of {@code annotationType} is either
+	 * <em>present</em> or <em>meta-present</em> on the test method belonging
+	 * to the specified {@code context}.
+	 */
+	public static boolean annotationPresentOnTestMethod(ExtensionContext context,
+			Class<? extends Annotation> annotationType) {
+		//@formatter:off
+        return context.getTestMethod()
+                .map(testMethod -> AnnotationSupport.isAnnotated(testMethod, annotationType))
+                .orElse(false);
+        //@formatter:on
```

Comment:
Oops, looks like the whitespace here is misaligned. ;)

Reply:
Good catch! Some spaces snuck in. :scream: